### DISCUSSION TITLE
Spatial 10X Visium gene expression integration to perform comparative analysis of across experimental conditions

### DISCUSSION THREAD

**Discussion by YannVRB** [satijalab/seurat#4366] [Q&A]
Hi,

First of all great job for the tutorial "Analysis, visualization, and integration of spatial datasets with Seurat".
I have also used a lot the tutorial "Introduction to scRNA-seq integration" and I was wondering if you were planning to post a similar tutorial using the Seurat tools to compare 10X visium datasets between two conditions (for example WT vs mutant).
I guess I am wondering how similar it would be with with the anchors and combined objects presented in the "Introduction to scRNA-seq integration". I basically want to perform differential gene expression analyses in a spatial dependent manner between my two conditions.

Thanks a lot in advance,
Yann

**Comment by nina-hahn** [comment]
Dear all, 

I am facing the same goal and would like to promote Yanns question. How should I handle biological replicats and 2 treatment groups (control vs treatment)?

Kind regards, 
Nina

**Comment by Umaarasu** [comment]
Hi @nina-hahn and @YannVRB ..did you solve this? how did you integrate?

**Comment by YannVRB** [comment]
I went with a third party platform called Partek. When uploading the SpaceRanger outputs on the platform, the integration is automatic. You can simply setup your condition at the beginning of the worflow and once you have identified your different clusters you can easily performed differential gene expression analysis.
I am aware that it is not the best for reproducibility because I cannot provide an actual R script for this analysis and Partek platform can be pretty expensive to buy a single user license, I apologize for that.
However, feel free to have a look at my two preprints that include two different Visium Spatial Gene Expression datasets.
https://www.biorxiv.org/content/10.1101/2023.01.18.524406v1 
https://www.biorxiv.org/content/10.1101/2023.01.18.524576v1

**Reply by Umaarasu** [reply]
Thanks :)
